16-lambda/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM amazon/aws-lambda-nodejs:16
```

Comment:
It looks like this upstream image doesn't exist

Reply:
Huh, AWS doesn't support the v16 runtime yet. I'll drop it from the PR. Thanks